solution_runner/commands/consts.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from pathlib import Path
+from zoneinfo import ZoneInfo
+
+import click
+
+
+FIRST_AOC_YEAR = 2015
+DECEMBER = 12
+ADVENT_DAYS_RANGE = click.IntRange(1, 25)
+ZERO = '0'
+INPUT_URL = 'https://adventofcode.com/{year}/day/{day}/input'
+SESSION = 'session'
+
+APP_DATA_DIRECTORY = click.get_app_dir('Advent of Code')
+CONFIGURATION_FILE_NAME = Path('configuration.yaml')
+ROOT_DIRECTORY_TYPE = click.Path(file_okay=False, dir_okay=True, writable=True, readable=True, resolve_path=True)
+ROOT_DIRECTORY = 'root directory'
+SOLUTION_PARTS = ('1', '2')
+SESSION_ID = 'session ID'
```

Comment:
Where is the user supposed to get it from?

Reply:
From the cookie stored in his browser.